compiler/compile/OMRSymbolReferenceTable.hpp

```diff
@@ -240,8 +240,8 @@ class SymbolReferenceTable
    TR::SymbolReference * getSymRef(CommonNonhelperSymbol i);
    TR::SymbolReference * getSymRef(int32_t i) { return baseArray.element(i); }
 
-   TR::SymbolReference * createRuntimeHelper(TR_RuntimeHelper, bool = false, bool = false, bool preservesAllRegisters = false);
```

Comment:
The only caller to createRuntimeHelper(), findOrCreateRuntimeHelper() always gives all parameters. In addition, the function looks a dedicated helper of findOrCreateRuntimeHelper() and should not be called by anyone else; and hence allowing createRuntimeHelper() have default parameters looks confusing to me.
For the second suggestion, I agree it makes the code easier to understand by having names with those bool parameters, I'll update them.